### DISCUSSION TITLE
RTSP stream to webRTC in Ant media server

### DISCUSSION THREAD

**Discussion by deekshithkv** [ant-media/Ant-Media-Server#5430] [Q&A]
Hi,
I am using RTSP stream from my IP camera as input to AntMedia server and streaming webRTC from the server. After testing it seems that AntMedia does not provide a webRTC URL to be played in player, instead it provides a stream ID which can only be played in player provided by AntMedia server. Is my observation correct? Is there any way by which i can configure Antmedia to give out a webRTC URL so that all players supporting webRTC streaming can use the same URL for streaming.

**Comment by USAMAWIZARD** [comment]
@deekshithkv WebRTC requires Signaling (some message sharing) for playing the video. So a direct URL for WebRTC is not possible. if you want to use your own custom player for playing with WebRTC Please take a look a look at our SDKs.
here is the [JS SDK](https://antmedia.io/docs/guides/developer-sdk-and-api/sdk-integration/javascript-sdk/) example you can refer.
[here](https://antmedia.io/docs/category/sdk-integration/) is the list of all the SDKs that we support.
